angularjs/angular-cookies.d.ts

```diff
@@ -37,4 +37,27 @@ declare module angular.cookies {
         remove(key: string, options?: any): void;
     }
 
+    ///////////////////////////////////////////////////////////////////////////
+    // CookieStoreService DEPRECATED
+    // see https://code.angularjs.org/1.2.26/docs/api/ngCookies/service/$cookieStore
+    ///////////////////////////////////////////////////////////////////////////

```

Comment:
:-1: 
This is a problem with this whole file but the comments on the interface should be using jsdoc comments. This is particularly important because this API was depreciated. That way the note shows up in IDEs.

Something like
```javascript
/**
 * CookieStoreService DEPRECATED
 * see https://code.angularjs.org/1.2.26/docs/api/ngCookies/service/$cookieStore
 */
interface ICookieStoreService {
```

I realize now that I should have left that interface in there for legacy...